### PR TITLE
Chris c

### PR DESCRIPTION
**Pull request by cjc061000**

### CODE REVIEW

File: src/assets/css/LabDetails.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+a.returnLink {
```
Comment (alishanM) [inline src/assets/css/LabDetails.css]: move this to LabDetails.vue

```
<style>
a.returnLink {
 text-decoration: none;
    color: black
} 
</style>
```

File: src/data/labs.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
       "to": "/labs/emr", 
       "id": 0,
       "img": "https://media.us.badgr.io/uploads/badges/issuer_badgeclass_053c8194-da0d-4c9f-bf7b-3c94047b2a4f.png",
+      "awardBadgeName": "emr",
```
Comment (alishanM) [inline src/data/labs.json]: Don't think we need this anymore if we can just use the ID to uniquely identify each lab

File: src/scripts/repository.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+export default {
+  createConn: function () {
+    return axios.create({
+      baseUrl
+      // headers:{"Authorization":"Bearer myTokenHere"}
+    })
+  },
+  getIssuerBadgeByBadgeName (badgeName) {
+    var newAxios = axios.create({ baseURL: baseUrl })
```
Comment (alishanM) [inline src/scripts/repository.js]: var -> let

File: src/scripts/repository.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    return newAxios.get(issuerBadgeDetailsResource + badgeName)
+  },
+  getIssuerBadges () {
+    var newAxios = axios.create({ baseURL: baseUrl })
```
Comment (alishanM) [inline src/scripts/repository.js]: var -> let

File: src/views/Home.vue [hunk 5]
```diff
@@ -127,20 +68,57 @@
...
+    }
+    this.getBadgesFromAPI()
+  },
+  methods: {
+    getBadgesFromAPI: function (event) {
+      Repository.getIssuerBadges()
+        .then(response => {
+          console.log('badger search response:', response)
+          // this timeout is just so we can see the spinner working
+          setTimeout(() => {
```
Comment (alishanM) [inline src/views/Home.vue]: this is fine for testing, but we don't want to slow down user experience by 2 seconds

File: src/views/LabDetails.vue [hunk 6]
```diff
@@ -34,27 +36,105 @@
...
+      badgeResult: {},
+      markdownSource: ''
+    }
   },
-  data: () => ({
-    lab: {}
-  })
+  methods: {
+    getBadgesFromAPI: function (event) {
+      var routeId = this.$route.params.id
```
Comment (alishanM) [inline src/views/LabDetails.vue]: var -> let (throughout file)

File: src/assets/css/LabDetails.css (hunk 1 above)
Comment (cjc061000) [reply src/assets/css/LabDetails.css]: done

File: src/data/labs.json (hunk 2 above)
Comment (cjc061000) [reply src/data/labs.json]: removed

File: src/scripts/repository.js (hunk 3 above)
Comment (cjc061000) [reply src/scripts/repository.js]: letted

File: src/scripts/repository.js (hunk 4 above)
Comment (cjc061000) [reply src/scripts/repository.js]: letted

File: src/views/Home.vue (hunk 5 above)
Comment (cjc061000) [reply src/views/Home.vue]: REMOVED

File: src/views/LabDetails.vue (hunk 6 above)
Comment (cjc061000) [reply src/views/LabDetails.vue]: turned to let

**Review by alishanM — APPROVED** [review]

**Pull request merged**